editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -2157,29 +2168,75 @@ function useSelectedGridItems(): ElementPath[] {
   )
 }
 
-const rulerMarkerIconSize = 12 // px
+const rulerMarkerIconSize = 11 // px
```

Comment:
this is something tangentially related, but maybe it'd be good to export this and use it in the SVG definitions (e.g. `${11 / scale}`.

Reply:
Fixed in https://github.com/concrete-utopia/utopia/pull/6678/commits/6425c8b185eac8790ce7e6277e16d2380609ccb2.